lib/Habrok.js

```diff
@@ -78,13 +78,17 @@ Habrok.prototype.request = function request(raw, opts) {
     } else if (errorCode === ECONNRESET || statusCode > 299) {
       if ((!err && self.RETRY_CODES.indexOf(statusCode) === -1) || attempt >= self.RETRIES) {
         debug('fail', req);
-
+        if (has(options, 'debugRequest')) options.debugRequest({ attempt });
         return reject(err || boom.create(statusCode, null, body));
       }
 
       return setTimeout(
         () => {
           debug('retry', raw);
+          if (has(options, 'onRetriableFailure')) {
+            const returnedError = err || boom.create(statusCode, null, body);
+            options.onRetriableFailure(returnedError);
+          }
```

Comment:
Don't be afraid of whitespace. Optimize for readability. Try putting a new line after the if statement, see how it feels.